### PR TITLE
Hacktoberfest : Modified README.md

### PR DESCRIPTION
**Pull request by deeepsig**
Fixed minor grammar issues.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -6,11 +6,11 @@ description: >-
 
 # Overview
 
-**Developers of all skill levels** use Battlesnake to learn new technologies, improve their programming skills, and have fun with friends, family, and colleagues.
+**Developers of all skill levels** use Battlesnake to learn about new technologies, improve their programming skills, and have fun with friends, family, and colleagues.
```
Comment (aurorawalker) [inline README.md]: The original copy here, to say "... learn new technologies..." is preferred over the edit. The word 'about' doesn't add anything, and in fact removed a bit of the strength/punchiness of the original statement. At Battlesnake, the goal is to actually learn new technologies, not just learn about them. ;)

Please remove this addition.

**Review by aurorawalker — CHANGES_REQUESTED** [review]
Small edit requested - the changes are less grammatical errors and more stylistic choices - but if you can adjust that one bit I left a comment on, I will merge the rest.

Thanks for reviewing the docs!

File: README.md (hunk 1 above)
Comment (deeepsig) [reply README.md]: Thanks @aurorawalker !!. I never thought about it this way. I'll change it and commit :)

**Comment by deeepsig** [comment]
Hey @aurorawalker thanks for reviewing!. I've made the changes :)

**Review by aurorawalker — APPROVED** [review]
Thanks for the adjustment! Merging.

**Pull request merged**